src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($minutes-hand-width);
+
+    height: $minutes-hand-hight;
```

Comment:
There seems to be a typo in the variable name '$minutes-hand-hight'. It should be '$minutes-hand-height' according to the standard naming convention.